src/Tracing/Cache/TraceableCacheAdapterTrait.php

```diff
@@ -35,7 +36,7 @@ trait TraceableCacheAdapterTrait
     /**
      * {@inheritdoc}
      */
-    public function getItem($key)
+    public function getItem(mixed $key): CacheItem
```

Comment:
The variance rules are reversed for arguments, so just by doing this we should be fine:
```suggestion
    public function getItem($key): CacheItem
```